src/network/networkd-link.c

```diff
@@ -261,6 +261,20 @@ static int link_enable_ipv6(Link *link) {
         return 0;
 }
 
+static bool link_is_bridge_port(Link *link) {
+
+        assert(link);
+
+        if (link->flags & IFF_LOOPBACK)
+                return false;
+
+        if (link->network)
+                if (link->network->bridge)
+                        return true;
```

Comment:
I think you want to return false here?

Also, would this:

```
        if (!link->network)
                return false;

        if (link->network->bridge)
                return false;
```

be better? Referenced link_lldp_rx_enabled().